course_info/static/course_info/js/controllers/PeopleController.js

```diff
@@ -255,6 +342,28 @@
         $scope.renderName = function(data, type, full, meta) {
             return full.profile.name_last + ', ' + full.profile.name_first;
         };
+        $scope.renderRemove = function(data, type, full, meta) {
+            // TODO - maybe make this a directive?  the isolate scope
+            //        in a directive complicates things.  anything has
+            //        to be better than this mess, though.
+            var registrarFed = /^.*feed$/.test(full.source);
+            var cell = '<div class="text-center">';
+            if (!registrarFed) {
+                cell += '<a href="" ng-click="confirmRemove(' +
+                        'dtInstance.DataTable.data()[' + meta.row + '])" ' +
+                        'data-sisid="' + full.user_id + '">';
+            }
+            cell += '<i class="fa fa-trash-o"';
+            if (registrarFed) {
+                cell += ' style="color: #ccc;"';
```

Comment:
probably not...i've been pretty lazy about leaving the styles from vittorio's sample code as inline.